mne_bids/commands/raw_to_bids.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
+# Authors: Teon Brooks  <teon.brooks@gmail.com>
+
+"""Command line interface for meg_bids.
+
+example usage:  $ mne_bids raw_to_bids --subject sub01 --task rest --out out_path
```

Comment:
Can you also add a line in the landing pages, i.e., `README.md` and `index.rst` ?